lib/line/bot/client.rb

```diff
@@ -547,6 +547,18 @@ def create_rich_menu(rich_menu)
         post(endpoint, endpoint_path, rich_menu.to_json, credentials)
       end
 
+      # Validate a rich menu
+      #
+      # @param rich_menu [Hash] The rich menu represented as a rich menu object
+      #
+      # @return [Net::HTTPResponse]
+      def validate_rich_menu(rich_menu)
```

Comment:
[must fix]

Unlike other endpoints, this endpoint validates an object, not a rich menu. So the name should be represented more accurately.

```suggestion
      def validate_rich_menu_object(rich_menu)
```